spec/models/player_spec.rb

```diff
@@ -0,0 +1,2 @@
+require 'rails_helper'
+
```

Comment:
Now you've got experience writing association tests in the model specs, in future try to test-drive associations by writing the association tests *before* adding the associations to the model.